README.md

```diff
@@ -372,13 +377,6 @@ For the JSON blob specification, see [`campaignSpec.md`](https://github.com/AdEx
 
 <div class='break-page'></div>
 
-#### Paying by impression (CPM) or by click (CPC)
```

Comment:
don't remove this, just add a clarification that it's not implemented. Remember this is the protocol document, not the DSp document. So anything possible at a protocol level should remain

Reply:
I removed it because now they will have the option to pay per impression or click, so the whole explanation that the default option is impression is not relevant, or do you mean something else with this?